pkg/controller/bootstrap/bootstrap.go

```diff
@@ -133,6 +135,21 @@ func (b *Bootstrap) Run(destDir string) error {
 	}
 	configs = append(configs, rconfigs...)
 
+	if cconfig.Spec.Proxy != nil && cconfig.Spec.Proxy.HTTPProxy != "" {
+		if err := os.Setenv("http_proxy", "cconfig.Spec.Proxy.HTTPProxy"); err == nil {
+			if err := podmanCreateUseProxy("k8s.gcr.io/pause:3.1"); err != nil {
+				return err
+			}
+		}
+	}
+	if cconfig.Spec.Proxy != nil && cconfig.Spec.Proxy.HTTPSProxy != "" {
+		if err := os.Setenv("https_proxy", "cconfig.Spec.Proxy.HTTPProxy"); err == nil {
```

Comment:
I think you will want to refactor this so the http or https proxy get set prior to running the podmanCreateUseProxy... The environment variables should be HTTP_PROXY and HTTPS_PROXY.

Make sure to remember to remove the quotes around cconffig.Spec>proxy.HTTPProxy

Reply:
These environment variables get pushed into the cmd.Env mentioned below